main.c

```diff
@@ -277,8 +315,9 @@ int main(void) {
   chSysInit();
 
   // Setup UART1
+  profiles[currentProfile](ledColors);
```

Comment:
keep the "Setup UART1" comment next to the UART1 setup section (line 320) (Or remove it it seems kind of redundant anyway)